hello-strongbox-maven/pom.xml

```diff
@@ -17,22 +20,22 @@
 
     <repositories>
         <repository>
-            <id>releases</id>
+            <id>strongbox-releases</id>
```

Comment:
But I see no benefit of this, unless you already have a `settings.xml` in which you already coincidentally have a

```
<server>
    <id>releases</id>
    ...
</server>
```

So... I don't see the benefit of this change.